intellij/pom.xml

```diff
@@ -49,6 +40,13 @@
             <systemPath>${project.basedir}/lib/commons-validator-1.4.1.jar</systemPath>
         </dependency>
 
+        <dependency>
+            <groupId>org.infernus.idea</groupId>
+            <artifactId>checkstyle</artifactId>
+            <version>5.18.5</version>
```

Comment:
The last published version is `5.23.0` (https://plugins.jetbrains.com/plugin/1065-checkstyle-idea). Maybe update while we are at it? You can update https://github.com/TestRoots/watchdog/blob/a01d99d34890334d8852a3a0ec9dc01445470a70/intellij/fetchidea.sh#L47 to the new version number to fix the checkstyle update.

I think that also resolves your previous issues, where the project was not building, because CheckStyle was missing. I think if you update the version number in the bash file, you can revert the changes here in the `pom.xml`. I should probably have added a catch of 404 for that in the bash file...